http/utility.hpp

```diff
@@ -584,7 +584,7 @@ inline std::string convertToAscii(const uint64_t& element)
     {
         return {};
     }
-    return std::string(bytearray.begin(), bytearray.end());
+    return {bytearray.begin(), bytearray.end()};
```

Comment:
I don't totally understand why clang thinks this is better. This is good code though, so I am good.